README.md

```diff
@@ -78,10 +78,10 @@ NPROCS=1               # Number of processes to use during compilation PER PLATF
 docker buildx build --platform $PLATFORMs --load --build-arg NODE_VERSION=$NODE_VERSION --build-arg NPROCS=$NPROCS -t static-node:$NODE_VERSION .
 ```
 
-Then, we extract the binary for a  for a specific platform:
+Then, we extract the binary for a specific platform:
 ```
 PLATFORM='linux/arm/v7'
-APP=$(docker run --platform $PLATFORM --rm -it -d static-node:$NODE_VERSION) 
-docker cp $APP:/usr/src/app/node/out/Release/node ./node-static-armv7-$PLATFORM-$NODE_VERSION
-docker kill $APP
+IN='/usr/src/app/node/out/Release/node'
+OUT=$(echo "node-static-$PLATFORM-$NODE_VERSION" | tr / -)
```

Comment:
The issue I came across here is that `./node-static-armv7-$PLATFORM-$NODE_VERSION` would expand to a string with slash characters and a hardcoded `-armv7` component, e.g. `node-static-armv7-linux/arm/v6-v12.22.3`, but the directories did not exist. I guess that the slash characters were not intentional, and above I propose `tr / -` to replace them with dashes. `OUT` becomes a string like `node-static-linux-arm-v6-v12.22.3`.

Reply:
I forgot to take care of those in the write up. Good catch.